x-pack/plugins/observability_solution/infra/public/pages/logs/log_entry_categories/sections/top_categories/single_metric_comparison.tsx

```diff
@@ -28,15 +27,15 @@ export const SingleMetricComparison: React.FunctionComponent<{
   } else if (changeFactor > 0 && Number.isFinite(changeFactor)) {
     return (
       <NoWrapSpan>
-        <EuiIcon type="sortUp" color="success" />
-        <EuiTextColor color="success">{formatPercentage(changeFactor)}</EuiTextColor>
+        <EuiIcon type="sortUp" color="accentSecondary" />
```

Comment:
Question: What's the use case of this? If the intention here is to convey success/confirmation then it should stay `success`. If it's used as brand color it should be `accentSecondary`. (see the section on the difference under "Color changes" in this [document](https://github.com/elastic/kibana/issues/199715))

Reply:
I think it can keep `success`, thanks